index.Rmd

```diff
@@ -119,8 +124,12 @@ walk2(
 d %>%
   ggplot(aes(y = fct_rev(Candidat), fill = fct_rev(`Date de publication`))) +
     geom_histogram(stat = "count") +
-    scale_fill_brewer(palette = "Greens", direction = -1) +
+    scale_fill_brewer(palette = "Greens", direction = 1) +
```

Comment:
Is this change on purpose?

Reply:
Woops, absolutely not. It appeared different on my laptop compared to the website.

Reply:
```suggestion
    scale_fill_brewer(palette = "Greens", direction = -1) +
```